### DISCUSSION TITLE
llama.dll getting fatter and fatter on windows?

### DISCUSSION THREAD

**Discussion by jrtp** [ggml-org/llama.cpp#8007] [General]
Hi there, awesome work! One question, I started compiling llama.cpp on windows with cuda a few weeks ago, back then the finished compile was around 36MB. Now it made a big jump and keeps growing, currently its at 80MB for the dll - does anybody know if that is correct behaviour? These are my compile steps:

set FORCE_CMAKE=1 && set "CMAKE_ARGS=-DLLAMA_CUBLAS=on -DLLAMA_AVX=off -DLLAMA_AVX2=off -DLLAMA_FMA=off"
python -m pip install .[all]

**Comment by slaren** [comment]
It is expected, the reason is that there are more kernels being compiled, which increases the size of the binary.
